R/cohort-diagnostics-cohort-overlap.R

```diff
@@ -518,9 +489,12 @@ cohortOverlapModule <- function(id,
         paste0("No cohort overlap data for this combination.")
       ))
 
+     plotHeight <- 300 * length(selectedDatabaseIds())
+      session$sendCustomMessage(ns("overlapPlotHeight"), sprintf("%spx", plotHeight))
+
       plot <- plotCohortOverlap(
         data = data,
-        shortNameRef = cohortTable,
+        cohortTable = cohortTable,
         yAxis = input$overlapPlotType
       )
       return(plot)
```

Comment:
The cohort overlap plots look unfinished and different from what I have seen in CD before. Is this intentional? The y-axis can use clearer labelling, the cohort IDs are not lined up on the left and right of the plot (only on the left right now), and hovering over the bars does not give very relevant/useful information.